CONTRIBUTING.md

```diff
@@ -120,14 +122,364 @@ rework your branch. You should rework any relevant information from the GitHub
+
+- Prepare your environment
+
+  > **Note:**: Most of the
+  > [Kata Containers repositories](https://github.com/kata-containers)
+  > contain code written in the
+  > [Go language (golang)](https://golang.org/). Go requires all code to be put
+  > inside the directory specified by the `$GOPATH` variable. Since this example PR
+  > is not using golang you do not need to
+  > [install golang](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components).
+  > However it still makes sense to put the code in the standard location.
+
+  ```sh
+  $ export GOPATH=${GOPATH:-$HOME/go}
+  $ mkdir -p "$GOPATH"
+  ```
+
+  > **Note:** The code above is safe to run whether you have golang installed
+  > or not.
+  > For further details on golang, refer to the
+  > [requirements section of the Kata Developer Guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components).
+
+
+- [Clone the upstream repository](https://help.github.com/articles/cloning-a-repository):
+
+  "Cloning" is a term that means to create a *local copy* of a Git repository
+  that generally lives on a remote server.
+
+  ```sh
+  $ dir="$GOPATH/src/github.com/kata-containers"
+  $ mkdir -p "$dir"
+  $ cd "$dir"
+  $ git clone https://github.com/kata-containers/community
+  $ cd community
+  ```
+
+  > **Note:** The previous Git command:
+  >
+  > - Creates a *local copy* of the upstream repository and switches to the
+  >   `master` *branch*.
+  > - Creates an "alias" (name) for the upstream repository URL called `origin`.
+
+  There are now *three* copies of the repository (all *exactly* the same):
+
+  - The original upstream one (https://github.com/kata-containers/community)
+
+    You do not (and cannot) make changes to this repository. PRs are a way of
+    allowing the project developers and administrators to review your work and
+    decide if they want to add your work to the upstream branch.
+
+    > **Note:** Don't worry - even if you make a mistake,
+    > you *cannot* do any damage to this repository.
+
+  - Your fork (`https://github.com/${your-github-username}/community`) on GitHub
+
+    This is where you send or upload (`git push`) your PR branches.
+
+  - Your *local copy* of the upstream repository
+    (https://github.com/kata-containers/community)
+
+    This is where you actually make changes to your private copy of the
+    repository.
+
+- [Setup a "git remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork):
+
+  "What happened to my fork?". The following demonstrates where your fork is
+  and how you use it.
+
+  Recall that your fork on the GitHub server is an *exact copy* of the
+  upstream repository that also lives on the GitHub server. That might not be
+  true moving forward because PRs are constantly merged into the upstream
+  repository. This means your fork will be out of date because is will not
```

Comment:
is / it 
This means your fork will be out of date because it will not

Reply:
Good catch @klynnrif - fixed!